Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -1,314 +1,311 @@
-
-```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE} 
-data_to_plot <- my.data[my.data$CMT==3,]
-data_to_plot$DAY_label <- paste("Day", data_to_plot$PROFDAY)
-data_to_plot$DAY_label[data_to_plot$DAY_label=="Day 0"] = "Baseline"
-data_to_plot <- data_to_plot[data_to_plot$DAY_label%in%c("Baseline","Day 2", "Day 4", "Day 6"),]
-
-gg <- ggplot(data = data_to_plot, aes(x=DOSE,y=LIDV, group = DOSE))+theme_bw()
-gg <- gg + stat_summary(geom = "errorbar", size = 1, 
-                        fun.data = function(y){
-                          y <- stats::na.omit(y)
-                          data.frame(
-                          y = mean(y),
-                          ymin = mean(y)-qt(0.975,length(y))*sqrt(stats::var(y)/length(y)), 
-                          ymax = mean(y)+qt(0.975,length(y))*sqrt(stats::var(y)/length(y)))
-                          })
-  gg <- gg + stat_summary(geom="point", fun.y=mean)
-gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
-gg <- gg + ylab("Continuous PD Marker (IU/L)") + xlab("Dose (mg)")
-gg + facet_grid(~DAY_label)
-```
-
-### Explore covariate effects on Dose-Response relationship
-
-```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE} 
-data_to_plot <- my.data[my.data$CMT==3,]
-data_to_plot$DAY_label <- paste("Day", data_to_plot$PROFDAY)
-data_to_plot$DAY_label[data_to_plot$DAY_label=="Day 0"] = "Baseline"
-data_to_plot <- data_to_plot[data_to_plot$DAY_label%in%c("Baseline","Day 6"),]
-
-
-gg <- ggplot(data = data_to_plot, aes(x=DOSE+0.1*DOSE*(SEX=="Female"),y=LIDV, color = SEX))+theme_bw()
-gg <- gg + stat_summary(geom = "errorbar", size = 1, 
-                        fun.data = function(y){
-                          y <- stats::na.omit(y)
-                          data.frame(
-                          y = mean(y),
-                          ymin = mean(y)-qt(0.975,length(y))*sqrt(stats::var(y)/length(y)), 
-                          ymax = mean(y)+qt(0.975,length(y))*sqrt(stats::var(y)/length(y)))
-                          })
-  gg <- gg + stat_summary(geom="point", fun.y=mean)
-  gg <- gg + stat_summary(geom="line", fun.y=mean)
-gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
-gg <- gg + ylab("Continuous PD Marker (IU/L)") + xlab("Dose (mg)")
-gg + facet_grid(~DAY_label)
-```
-
-## R Session Info
-```{r}
-sessionInfo()
+---
+title: "PD, Dose-Response - Continuous Endpoint"
+author: "Camille Vong"

```

Comment:
author: "Alison Margolskee, Camille Vong"

Reply:
convention: creating author first, followed by additional authors

Reply:
added now, thank you